### ISSUE TITLE
v1.6.0a2 learning_rate no longer logged

### ISSUE THREAD

**Issue by cxrodgers** [talmolab/sleap-nn#422]
I recently upgrade to v1.6.0a2 (`git checkout v1.6.0a2`). I notice that learning rate is no longer logged. I wonder if it may have to do with a name change from "learning_rate" to "train/lr" in #417 . This is a minor issue since I believe the quality of training is likely unaffected, but it would be nice to be able track the learning rate.

Example of my training_log.csv:
```
epoch,train/loss,val/loss,learning_rate,train/time,val/time
0,,0.006371453870087862,,,
0,,0.00023385054373648018,,,
1,0.0006624094676226377,0.0002221532049588859,,32.81522750854492,6.364217758178711
2,0.00021822562848683447,0.0002183269098168239,,28.42135238647461,6.391291618347168
3,0.00021450014901347458,0.00021747651044279337,,28.461719512939453,6.459515571594238
```
Note there is no entry in the fourth column. This is with a "centroid" output head but I think the issue is the same for other output heads as well. 

Also I note that there are two entries for epoch 0, both with val/loss and no train/loss, I think this issue may have existed in previous versions too.

Output of `sleap doctor` follows...

```
(sleap1.5) crodge4@bio:~/dev/sleap$ sleap doctor

SLEAP System Diagnostics
========================
Generated: 2026-01-19 12:22:14

[Platform]
  OS:        Linux 5.15.0-164-generic
  Platform:  Linux-5.15.0-164-generic-x86_64-with-glibc2.35
  Machine:   x86_64
  Processor: x86_64
  RAM:       54.3 GB / 503.5 GB
  Disk:      6.0 TB / 27.8 TB

[Python]
  Version:     3.13.0
  Executable:  /home/crodge4/miniconda3/envs/sleap1.5/bin/python3.13
  Prefix:      /home/crodge4/miniconda3/envs/sleap1.5

[Conda]
  Status:             ACTIVE
  Environment:        sleap1.5
  Prefix:             /home/crodge4/miniconda3/envs/sleap1.5
  Version:            conda 24.11.1
  auto_activate_base: False
  SLEAP in conda:     sleap, sleap-io, sleap-nn
  WARNING: Conda SLEAP packages may conflict with uv/pip

[GPU / CUDA]
  NVIDIA Driver: 570.169 (CUDA 12.8)
  GPU 0:         NVIDIA RTX A6000 (37442 MB free / 49140 MB)
  GPU 1:         NVIDIA RTX A6000 (37461 MB free / 49140 MB)
  GPU 2:         NVIDIA RTX A6000 (42525 MB free / 49140 MB)
  GPU 3:         NVIDIA RTX A6000 (20601 MB free / 49140 MB)
  PyTorch:       v2.8.0+cu128 (CUDA 12.8)

[Packages]
  sleap:                  v1.6.0a2 (local)
  sleap-io:               v0.6.3 (pip)
  sleap-nn:               v0.1.0a3 (pip)
  numpy:                  v2.2.6 (pip)
  h5py:                   v3.15.1 (pip)
  PySide6:                v6.10.1 (pip)
  opencv-python:          v4.12.0.88 (pip)
  opencv-python-headless: v4.12.0.88 (pip)
  torch:                  v2.8.0 (pip)
  imageio:                v2.37.2 (pip)
  imageio-ffmpeg:         v0.6.0 (pip)
  av:                     v15.1.0 (pip)

[CLI Binaries]
  sleap:
    Path:      /home/crodge4/miniconda3/envs/sleap1.5/bin/sleap
    Source:    conda
    Python:    /home/crodge4/miniconda3/envs/sleap1.5/bin/python3.13
  sleap-nn:
    Path:      /home/crodge4/miniconda3/envs/sleap1.5/bin/sleap-nn
    Source:    conda
    Python:    /home/crodge4/miniconda3/envs/sleap1.5/bin/python3.13
  sleap-nn-track:
    Path:      /home/crodge4/miniconda3/envs/sleap1.5/bin/sleap-nn-track
    Source:    conda
    Python:    /home/crodge4/miniconda3/envs/sleap1.5/bin/python3.13
  sio:
    Path:      /home/crodge4/miniconda3/envs/sleap1.5/bin/sio
    Source:    conda
    Python:    /home/crodge4/miniconda3/envs/sleap1.5/bin/python3.13
  ffmpeg:
    Path:      /usr/bin/ffmpeg
    Source:    PATH
    Python:    v4.4.2-0ubuntu0.22.04.1
  ffmpeg (imageio):
    Path:      /home/crodge4/miniconda3/envs/sleap1.5/lib/python3.13/site-packages/imageio_ffmpeg/binaries/ffmpeg-linux-x86_64-v7.0.2
    Source:    imageio-ffmpeg
    Python:    v7.0.2-static

[PATH (relevant entries)]
  /home/crodge4/miniconda3/envs/sleap1.5/bin
  /home/crodge4/miniconda3/condabin

Copy this output when reporting issues at:
https://github.com/talmolab/sleap/issues

Tip: Use sleap doctor -o to save diagnostics to a file
```
